docker/start_demo.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I'd put `start_demo.sh` and `stop_demo.sh` and others (like `Build`) in a single `Makefile`.

Reply:
Why would we do this? I think the current way seems to be the easiest way to let user get started.

Reply:
We will add more shortcuts like `start_demo.sh` and `stop_demo.sh` so there will be more files like these. Meanwhile, with the Makefile we can group all shortcuts (commands) in one single one.